index.js

```diff
@@ -39,6 +39,14 @@ module.exports = function (rawOptions) {
 						if (nodeChecked) {
 							var pathToInline = path.resolve(pathToCWD, nodeChecked);
 
+							if (path.isAbsolute(nodeChecked) && options.root) {
+								pathToInline = path.resolve(path.join(options.root, nodeChecked));
+							}
+
+							if (!fs.existsSync(pathToInline)) {
```

Comment:
I think I will remove this line because:

1. I would want to warn the user if the file does not exist.
2. `readFile` will reject the promise if the file is not found. This makes `existsSync` an anti-pattern:
   > Using `fs.exists()` to check for the existence of a file before calling `fs.open()`, `fs.readFile()` or `fs.writeFile()` is not recommended.
3. I would not like to use a synchronous method when we’re already returning a promise.